app/admin/teams.rb

```diff
@@ -0,0 +1,44 @@
+ActiveAdmin.register Team do
+  menu parent: 'Main App'
+
+  includes :organization
+  permit_params :name, :description, :organization_id, :team_members
+
+  index do
+    id_column
+    column :name
+    column :description
+    column :organization
+    column :created_at
+    column :updated_at
+    actions
+  end
+
+  show do
+    attributes_table do
+      row :id
+      row :name
+      row :description
+      row :organization
+      row :team_members
+      row :created_at
+      row :updated_at
+    end
+  end
+
+  filter :name
+  filter :description
+  filter :organization
+  filter :created_at
+  filter :updated_at
+
+  form do |f|
+    f.inputs do
+      f.input :name
+      f.input :description
+      f.input :organization_id, as: :select, collection: Organization.all,
+                                include_blank: false
```

Comment:
No estoy seguro, pero creo que es lo mismo que hacer 

```suggestion
      f.input :organization
```

Me queda la duda por el `include_blank: false` jeje. Si fuese lo mismo, quizás queda menos verboso así y lo mismo en `TeamMember`s y en `Employee`s

Qué opinas?